src/components/graph-view.js

```diff
@@ -761,12 +794,45 @@ class GraphView extends Component {
     // Merge
     nodes.enter().merge(nodes);
 
-    // Update All
-    nodes
-      .each(function(d, i, els) {
+    function updateNode(d, i, els) {
+      // setTimeout is used to unblock the browser
+      // clearing the previous render's timeout prevents the browser from being overworked
+      if (self.nodeTimeouts[`nodeKey-${d[nodeKey]}`]) {
```

Comment:
The weird thing is, using d[nodeKey] doesn't seem to work for some nodes. For instance, nodes with ID 24, 25, and 26 would render but 27 wouldn't. No explanation. Add the string `nodeKey` to convert the array to a map and everything works.